src/main/java/parser/Parser.java

```diff
@@ -21,9 +22,15 @@ public Command parseCommand(String[] input) {
             return new ByeCommand();
```

Comment:
Not sure why there is an extra indentation for 'return new ByeCommand()' here.
Minor typo probably!

Reply:
Amended!